packages/v4/src/content/design-guidelines/extensions/quick-start/quick-start.md

```diff
@@ -0,0 +1,103 @@
+---
+id: Quick start
```

Comment:
I noticed that the id here is "Quick start" and the section that I imagine this doc going is currently "Quick starts". I imagine we would want the latter? What do you think?

Reply:
+1 - these will need to match to avoid having 2 separate sections appear on the website

Reply:
Great catch, will update to "quick starts"!